python/lsst/jointcal/jointcal.py

```diff
@@ -157,16 +157,20 @@ def _build_ccdImage(self, dataRef, associations, jointcalControl):
 
         Returns
         ------
-        afw.image.TanWcs
-            the TAN WCS of this image
+        namedtuple
+            wcs : lsst.afw.image.TanWcs
+                the TAN WCS of this image, constructed from the calexp_md
+            key : (int, int)
+                a key to identify this dataRef by its visit and ccd ids
```

Comment:
I was going to argue that the key should be an opaque object, but I can see it being quite useful to have the parts named (may need to add "tract" or "filter" in the future, anyway). Done.